src/app/public/modules/repeater/repeater.component.spec.ts

```diff
@@ -1448,4 +1452,30 @@ describe('Repeater item component', () => {
       flush();
     }));
   });
+
+  describe('with nested repeater items', () => {
+    let fixture: ComponentFixture<NestedRepeaterTestComponent>;
+    let el: any;
+
+    beforeEach(fakeAsync(() => {
+      fixture = TestBed.createComponent(NestedRepeaterTestComponent);
+      el = fixture.nativeElement;
+      fixture.detectChanges();
+      tick();
+    }));
+
+    it('should reorder top-level repeater items', fakeAsync(() => {
+      const service = TestBed.inject(SkyRepeaterService);
+      const selector = 'sky-repeater-item-group-' + service.repeaterGroupId;
+      let items = el.querySelectorAll(selector);
+      const initialBottomRepeaterItem = items[items.length - 1];
+      el.querySelectorAll('.sky-repeater-item-reorder-top')[1].click();
+      fixture.detectChanges();
+      tick();
+      fixture.detectChanges();
+
+      const topRepeaterItem = el.querySelectorAll(selector)[0];
+      expect(topRepeaterItem).toBe(initialBottomRepeaterItem);
```

Comment:
This test does not work. Both `topRepeaterItem` and `initialBottomRepeaterItem` are undefined.

Reply:
Good catch! It was returning a false positive. My selector was wrong and the injected service wasn't quite right. I just hard coded the selector and fixed it up to work correctly.